src/schema/client.js

```diff
@@ -4,11 +4,12 @@
 import { blob, sqliteTable, text } from 'drizzle-orm/sqlite-core'
 import { dereferencedDocSchemas as schemas } from '@mapeo/schema'
 import { jsonSchemaToDrizzleColumns as toColumns } from './schema-to-drizzle.js'
-import { backlinkTable } from './utils.js'
+import { backlinkTable, customJson } from './utils.js'
 
 export const projectTable = sqliteTable('project', toColumns(schemas.project))
 export const projectBacklinkTable = backlinkTable(projectTable)
 export const projectKeysTable = sqliteTable('projectKeys', {
   projectId: text('projectId').notNull().primaryKey(),
   keysCipher: blob('keysCipher', { mode: 'buffer' }).notNull(),
+  projectInfo: customJson('projectInfo').default(JSON.stringify({})),
```

Comment:
```suggestion
  projectInfo: customJson('projectInfo').default({}),
```

Reply:
seems like there's a bug in Drizzle where this ideally would work but doesn't. i tried this in https://github.com/digidem/mapeo-core-next/pull/215/commits/bfe7c0f6f7cabf697d805781a3caaa27904e7096 but if you see the sql it generates, it doesn't do what you'd expect. have to keep the stringify here for now